sdks/python/apache_beam/runners/worker/opcounters.pxd

```diff
@@ -21,6 +21,22 @@ cimport libc.stdint
 from apache_beam.utils.counters cimport Counter
 
 
+cdef class TransformIoCounter(object):
```

Comment:
<!--new_thread; commit:4255543cdc7cfcadd29633f321599d2f591021d8; resolved:0-->
Let's capitalize IO in the name, i.e. `TransformIOCounter` (for consistency with, for example, `cStringIO` in the Python standard library).